libmscore/repeatlist.cpp

```diff
@@ -10,6 +10,7 @@
 //  the file LICENCE.GPL
 //=============================================================================
 
+#include <QtCore/QtDebug>
```

Comment:
sorry, please remove it then.  For some reason debug mode failed in eclipse when I first ran the original master branch.